src/components/NewItemButton/NewItemButton.styles.ts

```diff
@@ -0,0 +1,59 @@
+import { NewItemButtonState } from './NewItemButtonState.type';
+
+import type { BaseProps } from '@/types/BaseProps';
+
+export type NewItemButtonConfig = {
+  state?: Partial<Record<NewItemButtonState, BaseProps>>;
+  innerElements?: {
+    text?: BaseProps;
+  };
+} & BaseProps;
+
+export const defaultConfig = {
+  display: 'inline-flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  gap: '$space-component-gap-small',
+  h: '120px',
+  w: 'fit-content',
```

Comment:
The same as above.

Reply:
This would result in the button always occupying all available space. Should this be the case?

<img width="787" alt="image" src="https://github.com/VirtusLab/tetrisly-react/assets/160858266/62b477ec-1ae1-4575-b59d-b69374025c39">

Reply:
Yep, it's correct ;)